docs/usage/shoot_operations.md

```diff
@@ -1,3 +1,7 @@
+---
+title: Trigger Shoot Operations Through Annotations
```

Comment:
```suggestion
title: Trigger Shoot Operations
```

Reply:
I chose this title because I think for a beginner-ish user `Trigger Shoot Operations` reads as something potentially requiring a lot of steps/complexity. While adding ` Through Annotations` it suggests it's simplicity and reduces the barrier of clicking on the document.